dist/debian/debian/control

```diff
@@ -1,47 +1,47 @@
-Source: {{product}}-server
+Source: scylla-server
 Maintainer: Takuya ASADA <syuu@scylladb.com>
 Homepage: http://scylladb.com
 Section: database
 Priority: optional
 X-Python3-Version: >= 3.4
 Standards-Version: 3.9.5
 
-Package: {{product}}-conf
+Package: scylla-conf
```

Comment:
But we still need to make this work for enterprise. Can `control` not see DEB_BUILD_OPTIONS?